packages/esm-shr-app/src/community-referrals/patient-chart/referral-chart-view.component.tsx

```diff
@@ -1,84 +1,87 @@
+    return patient?.identifier.find((item) => {
+      return item.type.coding.some((coding) => coding.code === nationalPatientUniqueIdentifier);
+    })?.value;
+  };
 
+  const nationalPatientUniqueNumber = getPatientNupiNumber(patient, nationalPatientUniqueIdentifier);
+
+  const { referral, isLoading, isValidating, isError } = useCommunityReferral(nationalPatientUniqueNumber);
   const headerTitle = t('referral', 'Referrals');
 
-  if (isLoading) {
-    return <DataTableSkeleton rowCount={5} />;
-  }
-  if (referral == null) {
+  if (!nationalPatientUniqueNumber) {
     return (
       <Layer>
         <Tile className={styles.tile}>
           <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
             <h4>{headerTitle}</h4>
           </div>
           <EmptyDataIllustration />
-          <p className={styles.content}>There is no referral data to display for this patient.</p>
+          <p className={styles.content}>
+            {t(
+              'missingNupiIdentiferMessage',
+              'The patient is missing  national patient unique identifier (NUPI). Please register the patient with the client registry to retrieve their referrals.',
+            )}
+          </p>
         </Tile>
       </Layer>
     );
   }
-  return (
-    <Layer>
-      <Tile>
-        <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
-          <h4>{headerTitle}</h4>
-        </div>
-        <div className={styles.container}>
-          <div className={styles.content}>
-            <p>{t('referralDate', 'Referral Date')} </p>
-            <p>
-              <span className={styles.value}>{formatDate(new Date(referral?.referralReasons?.referralDate))}</span>
-            </p>
-          </div>
-          <div className={styles.content}>
-            <p> {t('status', 'Status')}</p>
-            <p>
-              <span className={styles.value}> {referral?.status} </span>
-            </p>
-          </div>
 
-          <div className={styles.content}>
-            <p> {t('ReasonCode', 'Reason Code')}</p>
-            <p>
-              <span className={styles.value}> {referral?.referralReasons?.reasonCode} </span>
-            </p>
-          </div>
-          <div className={styles.content}>
-            <p> {t('clinicalNote', 'Clinical Note')}</p>
-            <p>
-              <span className={styles.value}> {referral?.referralReasons?.clinicalNote} </span>
-            </p>
-          </div>
-          <div className={styles.content}>
-            <p> {t('referredFrom', 'Referred From')}</p>
-            <p>
-              <span className={styles.value}> {referral?.referredFrom} </span>
-            </p>
-          </div>
-        </div>
-      </Tile>
-    </Layer>
+  if (isLoading) {
+    return <DataTableSkeleton rowCount={5} />;
+  }
+
+  if (referral === null) {
+    return <EmptyState displayText={t('emptyReferralsMessage', ' referral data')} headerTitle={headerTitle} />;
```

Comment:
It is here which you should update 
```suggestion
    return <EmptyState displayText={t('emptyReferralsMessage', 'No referral data')} headerTitle={headerTitle} />;
```